package.json

```diff
@@ -227,8 +227,8 @@
   },
   "private": true,
   "dependencies": {
-    "@department-of-veterans-affairs/formation": "^6.8.0",
-    "@department-of-veterans-affairs/formation-react": "^5.0.2",
+    "@department-of-veterans-affairs/formation": "^6.9.0",
+    "@department-of-veterans-affairs/formation-react": "^5.1.0",
```

Comment:
Context - The updates are to add support for the PromoBanner to have text that wraps to the next line.